packages/front-end/components/ProtectedPage.tsx

```diff
@@ -170,6 +175,8 @@ const ProtectedPage: React.FC<{
       userAgent: window.navigator.userAgent,
       url: router?.pathname || "",
       cloud: isCloud(),
+      freeSeats: orgData?.data?.organization?.freeSeats || "",
```

Comment:
@jdorn I added these attributes to the `setAttributes` method with the idea that we can use them with the feature flag rules so we can target new users vs grandfathered users. Let me know if you think there is a better way to do this - I don't love that I had to introduce another API call.